ocp_resources/performance_profile.py

```diff
@@ -0,0 +1,121 @@
+from ocp_resources.resource import Resource
+
+
+class PerformanceProfile(Resource):
+    api_group = Resource.ApiGroup.PERFORMANCE_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        additional_kernel_args=None,
+        cpu=None,
+        globally_disable_irq_load_balancing=None,
+        hugepages=None,
+        machine_config_label=None,
+        machine_config_pool_selector=None,
+        net=None,
+        node_selector=None,
```

Comment:
`node_selector` is already part of Resource (part of kwargs); no need to specify explicitly

Reply:
ok.